services/shotgrid_common/utils.py

```diff
@@ -14,11 +15,50 @@
 from ayon_api.entity_hub import ProjectEntity
 from ayon_api.utils import slugify_string
 from ayon_api import get_attributes_for_type
+import logging
 
-from nxtools import logging
 import shotgun_api3
 
 
+def get_logger(name: str) -> logging.Logger:
+    """Return a logger instance with the given name."""
+    # get environment variable DEBUG level
+    log_level = os.environ.get("LOGLEVEL", "INFO").upper()
+
+    logger = logging.Logger(name)
```

Comment:
Avoid reinitialization of the same logger...
```suggestion

_loggers = {}


def get_logger(name: str) -> logging.Logger:
    """Return a logger instance with the given name."""
    if name in _loggers:
        return _loggers[name]

    # get environment variable DEBUG level
    log_level = os.environ.get("LOGLEVEL", "INFO").upper()

    logger = logging.Logger(name)
    _loggers[name] = logger
```